cmd/main.go

```diff
@@ -172,6 +172,11 @@ func main() {
 		NSXClient: nsxClient,
 		NSXConfig: cf,
 	}
+	err = commonService.NSXClient.ValidateLicense(true)
```

Comment:
i go through the method invoked inside, i only see one INFO level log indicate license changed, the others are all error logging, could we add some DEBUG level log to get more detail info?

Reply:
done